Actions/IncrementVersionNumber/IncrementVersionNumber.ps1

```diff
@@ -29,12 +29,30 @@ try {
     $telemetryScope = CreateScope -eventId 'DO0076' -parentTelemetryScopeJson $parentTelemetryScopeJson
 
     $settings = $env:Settings | ConvertFrom-Json
+    if ($versionNumber.StartsWith('+')) {
+        # Handle incremental version number
+        $allowedIncrementalVersionNumbers = @('+1', '+0.1')
```

Comment:
This looks like code duplication from `Set-VersionInSettingsFile`. Consider doing the check once.

Reply:
It was moved here to do the check once - probably just forgot to remove it in the other function

Reply:
It is really just a defensive check and if I remove it, then a number of tests are failing.